qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -5,33 +5,64 @@ weight: 6
 
 # How to Generate ColBERT Multivectors with FastEmbed
 
-With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+## ColBERT
 
-Please note that ColBERT requires more resources than other no-interaction models. We recommend you use ColBERT as a re-ranker instead of a first-stage retriever. 
+ColBERT is an embedding model that produces a matrix (multivector) representation of input text; it generates one vector per token (a `token` is a meaningful text unit for a machine learning model). 
+This embedding way allows ColBERT to capture deeper input semantics than many dense embedding models, which embed a whole input into just a single vector. 
+However, at the same time, storing multiple vectors per input usually leads to higher resource consumption, such as increased memory usage.
 
-The first-stage retriever can retrieve 100-500 examples. This task would be done by a simpler model. Then, you can rank the leftover results with ColBERT.
+So, even if ColBERT can be a powerful retriever, storing multivectors for all the entities in your data might be too costly. Therefore,
+we recommend using it mainly for reranking on a small retrieved set of data rather than first-stage retrieval. A simple dense retriever can retrieve around 100-500 examples at the first stage; 
+then, you can rerank them using ColBERT, moving the most relevant results to the top.
```

Comment:
```suggestion
Despite ColBERT being a strong retriever, it's speed limitation might make it less suitable for large-scale retrieval tasks. As a result, we recommend using ColBERT primarily for reranking a small set of already retrieved examples, rather than for first-stage retrieval. A simple dense retriever can initially retrieve around 100-500 candidates, which can then be reranked with ColBERT to bring the most relevant results to the top
```